ext/intl/calendar/calendar_methods.cpp

```diff
@@ -395,11 +403,7 @@ U_CFUNC PHP_FUNCTION(intlcal_set)
 	}
 
 	for (int i = 0; i < arg_num; i++) {
-		if (args[i] < INT32_MIN || args[i] > INT32_MAX) {
-			intl_error_set(NULL, U_ILLEGAL_ARGUMENT_ERROR,
-				"intlcal_set: at least one of the arguments has an absolute value that is too large", 0);
-			RETURN_FALSE;
-		}
+		ZEND_VALUE_ERROR_OUT_OF_BOUND_VALUE(args[i], i);
```

Comment:
Should this be `i + 1`? Argument numbers are 1-based an i is zero based.

Reply:
Indeed, good catch!